pyproject.toml

```diff
@@ -49,3 +44,6 @@ sections = "FUTURE,STDLIB,TYPING,THIRDPARTY,FIRSTPARTY,LOCALFOLDER"
 
 [tool.pytest.ini_options]
 addopts = "--doctest-modules"
+
+[dependency-groups]
+dev = ["pytest ~= 8.2.1"]
```

Comment:
This is an incomplete duplication of `[project.optional-dependencies]` above. I *think* the `dev` dependency group is the better way but we should not mix both. Honestly I'd just keep it as it was, and some day actually migrate this project to UV where this cleanup could be part of.